### ISSUE TITLE
Logging improvement

### ISSUE THREAD

**Issue by MikeLush** [jonnyhuck/RFCL-PolygonDivider#6]
Currently where a polygon cannot be divided the following message is logged:

> There was an un-dividable polygon in this dataset.

However, it's not possible to see from this which polygon has failed, which can make them difficult to track down in large, complex datasets. Would it be possible to add some details of the polygon, e.g. the coordinates of the centroid?

**Comment by jonnyhuck** [comment]
Yes - that is a good idea, an earlier iteration actually output an error shapefile, but logging some more information about where the failure took place would be very useful - I'll take a look at that.
